### ISSUE TITLE
I'm using Mask R-CNN matterpot for Lane Detection, And while training I'm getting error.

### ISSUE THREAD

**Issue by ShailendraLowanshi24** [matterport/Mask_RCNN#2830]
[/content/Mask_RCNN/mrcnn/model.py](https://localhost:8080/#) in train(self, train_dataset, val_dataset, learning_rate, epochs, layers, augmentation, custom_callbacks, no_augmentation_sources)
   2372             max_queue_size=100,
   2373             workers=workers,
-> 2374             use_multiprocessing=True,
   2375         )
   2376         self.epoch = max(self.epoch, epochs)

[/usr/local/lib/python3.7/dist-packages/keras/legacy/interfaces.py](https://localhost:8080/#) in wrapper(*args, **kwargs)
     85                 warnings.warn('Update your `' + object_name +
     86                               '` call to the Keras 2 API: ' + signature, stacklevel=2)
---> 87             return func(*args, **kwargs)
     88         wrapper._original_function = func
     89         return wrapper

[/usr/local/lib/python3.7/dist-packages/keras/engine/training.py](https://localhost:8080/#) in fit_generator(self, generator, steps_per_epoch, epochs, verbose, callbacks, validation_data, validation_steps, class_weight, max_queue_size, workers, use_multiprocessing, shuffle, initial_epoch)
   2009                 batch_index = 0
   2010                 while steps_done < steps_per_epoch:
-> 2011                     generator_output = next(output_generator)
   2012 
   2013                     if not hasattr(generator_output, '__len__'):

StopIteration:

**Comment by ShailendraLowanshi24** [comment]
Please help me

**Comment by ShailendraLowanshi24** [comment]
![error](https://user-images.githubusercontent.com/82945328/169107972-ffda7f2a-37ce-4595-9df8-0227cac5bd83.png)
![error](https://user-images.githubusercontent.com/82945328/169108000-d54d102d-7f34-4b6c-84e2-cf41ae9ef78e.png)
